src/dra/bss.c

```diff
@@ -1,461 +1,470 @@
+s32 D_80137E5C;
+s32 D_80137E60;
+s32 g_MemCardRStep;
+s32 g_MemCardRStepSub;
+s32 D_80137E6C;
+SVECTOR D_80137E70[14];
+s32 D_80137EE0;
+s32 D_80137EE4;
+s32 D_80137EE8;
+s32 D_80137EEC;
+s32 D_80137EF0;
+s32 D_80137EF4;
+padding u8 D_80137EF8[0x60];
+
+// cd.c
+CdCallbacks g_CdCallback;
+CdlLOC g_CdLoc;
+CdThing g_Cd;
+s16 g_VabId;
+u8 D_80137F96;
+padding u8 D_80137F97[5];
+s32 D_80137F9C;
+s32 D_80137FA0;
+padding u32 D_80137FA4;
+s16 D_80137FA8;
+padding s16 D_80137FAA;
+s32 D_80137FAC;
+s32 D_80137FB0;
+
+// 692E8.c
+s32 D_80137FB4;
+s32 D_80137FB8;
 #if defined(VERSION_US)
-u32 D_80137FBC;
+s32 D_80137FBC;
 #endif
-u16 g_WasFacingLeft[2];
-u16 g_WasFacingLeft2[2];
-u32 g_WasFacingLeft3;
-u32 g_WasFacingLeft4;
-u16 g_WasFacingLeft5[2];
-u32 g_WasFacingLeft6;
-u32 g_WasFacingLeft7;
+
+// 6D59C.c, 704D0.c
+s16 g_WasFacingLeft;
+padding s16 g_WasFacingLeft_;
+s16 g_WasFacingLeft2;
+padding s16 g_WasFacingLeft2_;
+s32 g_WasFacingLeft3;
+s32 g_WasFacingLeft4;
+s16 g_WasFacingLeft5;
+padding s16 g_WasFacingLeft5_;
+s32 g_WasFacingLeft6;
+s32 g_WasFacingLeft7;
+
+// 71830.c
 #if defined(VERSION_US)
-u32 D_80137FDC;
+s32 D_80137FDC;
+#endif
+
+// 72BB0.c, 75F54.c
+s32 D_80137FE0;
+s32 D_80137FE4;
+s32 D_80137FE8;
+s32 D_80137FEC;
+s32 D_80137FF0;
+
+// 75F54.c
+s32 g_WingSmashButtonCounter;
+s32 g_WingSmashButtonTimer;
+s32 g_WingSmashTimer;
+s32 g_BatScreechDone;
+s32 g_MistTimer; // remaining time in mist transformation
+s32 D_80138008;
+s32 D_8013800C[11];
+s32 D_80138038;
+
+// 75F54.c, 7A4D0.c
+u8 D_8013803C;
+padding u8 D_8013803C_[3];
+u8 D_80138040;
+padding u8 D_80138040_[3];
+u8 D_80138044;
+padding u8 D_80138044_[3];
+u8 D_80138048;
+padding u8 D_80138048_[3];
+
+// 78D0C.c
+Point16 D_8013804C[16];
+
+// 7E4BC.c
+s32 D_8013808C;
+s32 D_80138090;
+mistStruct D_80138094[16];
+s16 D_801381D4;
+padding s16 D_801381D4_;
+s16 D_801381D8;
+padding s16 D_801381D8_;
+s16 D_801381DC;
+padding s16 D_801381DC_;
+s16 D_801381E0;
+padding s16 D_801381E0_;
+s16 D_801381E4;
+padding s16 D_801381E4_;
+s16 D_801381E8;
+padding s16 D_801381E8_;
+s16 D_801381EC;
+padding s16 D_801381EC_;
+s16 D_801381F0;
+padding s16 D_801381F0_;
+Primitive D_801381F4[8];
+s32 D_80138394;
+s32 D_80138398;
+
+// 843B0.c
+Point16 D_8013839C[32];
+s32 D_8013841C;
+padding s32 D_80138420;
+RECT D_80138424;
+
+// 8A0A4.c, 8D3E8.c
+s32 D_8013842C;
+s32 D_80138430;
+padding u32 D_80138434;
+s32 D_80138438;
+s32 D_8013843C;
+s32 D_80138440;
+s32 D_80138444;
+s32 D_80138448;
+s32 D_8013844C;
+s32 D_80138450;
+
+// 91EBC.c, 92F60.c, 93290.c, 93BDC.c, 94F50.c, 953A0.c
+s32 D_80138454;
+s16 g_CurrentXaConfigId;
+padding s16 g_CurrentXaConfigId_;
+s16 D_8013845C;
+padding s16 D_8013845E;
+#if defined(VERSION_PC)
+u8 g_SeqTable[SS_SEQ_TABSIZ * SEQ_TABLE_S_MAX * SEQ_TABLE_T_MAX];
+u_long* D_80138784[0x700];
+#else
+u8 g_SeqTable[800];
+u_long* D_80138784[488];
+#endif
+s32 g_CurCdPos;
+u8 g_CdMode[2];
+padding u8 D_80138F26[2];
+s32 g_KeyOffChannels;
+u8 g_CdCommandResult[56];
```

Comment:
Length seems too long but it won't hurt anything
<img width="549" alt="image" src="https://github.com/user-attachments/assets/efb12ac5-077a-48de-85a3-caace1e5d038">